backend/open_webui/models/tools.py

```
@@ -85,115 +67,183 @@ class ToolResponse(BaseModel):
         ]
 
-    def get_tool_valves_by_id(self, id: str) -> Optional[dict]:
+    def get_tool_valves_by_id(
+        self, id: str, db: Optional[Session] = None
+    ) -> Optional[dict]:
         try:
-            with get_db() as db:
+            with get_db_context(db) as db:
                 tool = db.get(Tool, id)
                 return tool.valves if tool.valves else {}
         except Exception as e:
```

Ruff (0.15.1)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)